src/docker-event-monitor.go

```diff
@@ -75,6 +76,9 @@ func init() {
 		if len(glb_arguments.MailTo) == 0 {
 			log.Fatalln("E-Mail notification enabled. Recipient address required!")
 		}
+		if len(glb_arguments.MailUser) == 0 {
```

Comment:
I would not make the username a requirement. I tried with my gmail account and the SMTP username is totally irrelevant - all that is checked is the `MailFrom`. Therefore I would suggest to set it to what is set as `MailFrom`if it's empty.